### ISSUE TITLE
Site: Feedbacks for Color page

### ISSUE THREAD

**Issue by saadmhybris** [SAP/fundamental#47]
@saadmhybris 

- [x] 1. Side margin of the body
- [x] 2. Weight and color of the title
- [ ] 3. The intro text takes only 6 col
- [ ] 4. Color blocks: margin between the subtitle to the block
- [ ] 5. Color blocks: size of the block and text spacing in the block
- [ ] 6. Color blocks: Vertical spacing between blocks
- [ ] 7. Accessibility: Subtitle, weight and color
- [ ] 8. Accessibility: The whole section is 6 col
- [x] 9. Accessibility: there’s a link in the intro text “WCAG AA standards”
- [ ] 10. Accessibility: Spacing between intro text and subtitle
- [ ] 11. Accessibility: spacing between subtitle and text block
- [ ] 12. Accessibility: spacing between text block and next subtitle
- [x] 13. Accessibility: spacing between last text block and the footer
- [x] 14. Footer: “Have a question…GitHub.” should be in one line
- [x] 15. Footer: The link color is purple

![feedback_color page](https://user-images.githubusercontent.com/32305332/34731006-9b8b32ba-f526-11e7-8df3-611198c95cba.png)
